.github/workflows/validate-pr-actions.yaml

```diff
@@ -0,0 +1,108 @@
+---
+name: Validate Github Actions
+
+on:
+  workflow_call:
+    inputs:
+      image:
+        type: string
+        required: false
+  pull_request:
+    paths:
+      - actions/**
+      - tests/test_resources/mpyl_config.yml
+      - tests/test_resources/repository/changed_files.json
+      - tests/projects/job/**
+
+# If multiple commits are pushed simultaneously, only build the last one.
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number }}
+  cancel-in-progress: true
+
+jobs:
+
+  health-check:
+    name: ▶️ Run actions/health-check
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4.2.1
+
+      - name: Point action.yaml to a new version
+        if: inputs.image != ''
```

Comment:
Hm so on the pr-trigger we'll always use the version that is hardcoded? Shouldn't it point to the version (branch) of the pr in that case? Also to avoid hardcoding a version (see later comment)

Reply:
This workflow runs in two ways:
* [triggered by itself](https://github.com/Vandebron/gh-mpyl/pull/26/files/bf43f6b4e8e886bf9e6195486bd737e5336194a5#diff-1dab4111f843ddf950834c5edec3ac2aa7ce66f48125a4132fd725656ab10c92R10-R15) → tests the currently released currently released
* [triggered by `workflows/validate-pr-docker`](https://github.com/Vandebron/gh-mpyl/pull/26/files/bf43f6b4e8e886bf9e6195486bd737e5336194a5#diff-1dab4111f843ddf950834c5edec3ac2aa7ce66f48125a4132fd725656ab10c92R5) → tests a new Docker image produced by this PR (if a new image is produced)

Reply:
Ah sorry, I knew about the 2 ways but I didn't study the path's for way #1 closely enough. It only runs for action and config changes, so there is no need to use a new docker image since those are files being passed from the repo to the image. Bit of a mind-twister.